### ISSUE TITLE
Interested in having a homebrew formula?

### ISSUE THREAD

**Issue by ianks** [jimeh/tmuxifier#57]
I think it would be useful, and rather easy to implement. I would be happy to make this happen.

**Comment by jimeh** [comment]
Yes, homebrew, has been in the back of my mind for some time. #15 was a first step enabling the binary to be executed without any complicated environment setup.

But I still need to figure out what to do with the default layouts path. Initial thought is to set it to `$HOME/.tmuxifier-layouts` when installed via a package manager, and leave it pointing to within Tmuxifier's own folder when installed via git clone to keep things backwards compatible. Or alternatively just  bump the version to 2.0 and set the default to `$HOME/.tmuxifier-layouts` regardless of installation path. It's a choice of potential confusion vs likely annoyance to users.

I'd like to hear your thoughts on if you don't mind.

**Comment by vincecima** [comment]
@jimeh interest in still doing this?

**Comment by ArgonQQ** [comment]
Would highly appreciate this!
